### ISSUE TITLE
Sidekiq warning about serializing job arguments

### ISSUE THREAD

**Issue by jesseminer** [lardawge/carrierwave_backgrounder#304]
Hi, I'm using Carrierwave Backgrounder and Sidekiq. Sidekiq 6.4.1 prints the following warning when backgrounder enqueues a job:
> Job arguments to CarrierWave::Workers::ProcessAsset do not serialize to JSON safely. This will raise an error in Sidekiq 7.0. See https://github.com/mperham/sidekiq/wiki/Best-Practices or raise an error today by calling `Sidekiq.strict_args!` during Sidekiq initialization.

I think the only issue is that Carrierwave's `mounted_as` returns a symbol, and it needs to be converted to a string before it's passed off to Sidekiq. `mounted_as` is passed to `enqueue_for_backend` in [lib/backgrounder/orm/base.rb](https://github.com/lardawge/carrierwave_backgrounder/blob/master/lib/backgrounder/orm/base.rb).

**Comment by waqasnazir-gr** [comment]
@jesseminer Can you please carrierwave backgrounder version you install and where you set call back for backgrounder

**Comment by jesseminer** [comment]
@waqasnazir-gr My gemfile loads carrierwave backgrounder from github:
```
gem 'carrierwave_backgrounder', github: 'lardawge/carrierwave_backgrounder'
```

And an example of an active record model that uses the gem:
```
class Group < ApplicationRecord
  mount_uploader :logo, GroupLogoUploader
  process_in_background :logo
end
```

Let me know if you need any more info!

**Comment by waqasnazir-gr** [comment]
@jesseminer thanks! One more thing please share your carrierwave version this is not working for me with
 gem 'carrierwave', '~> 2.0'

**Comment by jesseminer** [comment]
@waqasnazir-gr I'm on carrierwave 2.1.1.

**Comment by holstvoogd** [comment]
I've made a PR that fixes this, forgot to link the issue though 😅 
#307

**Comment by lardawge** [comment]
This has been fixed and released in version 0.4.3
